### ISSUE TITLE
[Suggestion] Allow Multiple inputs to a switch/button

### ISSUE THREAD

**Issue by MrBester** [tsunamayo/Skywanderers#854]
With the recent logic update there have been many changes to the way things can be linked, mainly that buttons and switches can have only 1 input. this makes it very long winded to make something as simple as a multi button toggle for things such as doors or canopies. the way things work now is we have to chain buttons together to make something as simple as a door toggle. where as in the past we just connected all the buttons to the 1 switch and that toggled the door/hinge/rotor. is it possible to bring this feature back. im not a programmer so i don't know if the new system is intentional for performance. however i being a pleb would have thought it would be better for performance if the game only had to calculate sending the packet from button to switch rather than calculating the packet traveling from button, to button, to button, to switch for example.

**Comment by tsunamayo** [comment]
Yes, actually the old behaviour was to give an implicit OR - I choose to not redo it as the logic will work a bit differently now. But I could take it back, I will think about the pro and cons a bit more. In such a case no worry about performance.

**Comment by ComputerCake** [comment]
A workaround for now is to connect all the input buttons to an OR gate, and then that into the switch. This caught me off guard a bit too, I would like to be able to connect multiple buttons to one switch. In any case, this does work and stops the daisy-chaining.
